loleaflet/src/layer/tile/TilesSection.ts

```diff
@@ -164,7 +164,10 @@ class TilesSection {
 			this.context.fillStyle = this.containerObject.getClearColor();
 			this.context.fillRect(offset.x, offset.y, ctx.tileSize.x, ctx.tileSize.y);
 		}
-
+		if (L.Map.THIS.getTileSectionMgr() && L.Map.THIS.getTileSectionMgr()._zoomChanged) {
+			this.context.clearRect(offset.x, offset.y, ctx.tileSize.x, ctx.tileSize.y);
+			this.context.fillRect(offset.x, offset.y, ctx.tileSize.x, ctx.tileSize.y);
```

Comment:
This function "paintSimple" only takes care of tile painting for non-Calc apps. For calc it uses paintWithPanes() which handles calc case with freeze panes. So you need a similar change there too.